index.js

```diff
@@ -0,0 +1,49 @@
+const hamburgerBtn = document.getElementById("hamburger-btn");
+const closeBtn = document.getElementById("close-btn");
+const mobileMenu = document.querySelector(".nav-menu");
+
```

Comment:
Great job on the mobile menu! I have a few friendly suggestions that could enhance the user experience and meet all the project requirements:

- [ ] I noticed that when clicking on a link inside the hamburger menu, it doesn't redirect to the correct part of the page as requested in the requirements. Maybe we could work on fixing that.

- [ ] Additionally, it would be fantastic if the menu could automatically close after clicking on a link inside it, making navigation smoother for users.

- [ ] Regarding the design, could you please make some adjustments to the hamburger button so that it matches the template better? That would add a nice touch to the overall look.

   ![image](https://github.com/RolandAmini/portfolio-mobile/assets/107952724/d8d758a7-c230-4ee8-9d2b-3d53cfd12aee)
   
   **_Video for a better understanding._**
   
   [scrnli_8_2_2023_10-52-26 AM.webm](https://github.com/RolandAmini/portfolio-mobile/assets/107952724/6e944ccb-3a4b-4710-b9c8-ab37e2a396a4)

Reply:
thank you @taldr27 for the review,  all required changes have been implemented